benchmark/README.md

```diff
@@ -1,21 +1,60 @@
 # Comparing benchmarks for regression
 
-Build and run the benchmark executable with `--csvraw=results.csv` option for
-the baseline code i.e. without the changes.
+The following commands are tested with `cabal` version 3.0.
+
+Run the benchmarks for the baseline code i.e. without the changes:
 
 ```
-$ cabal v2-run bench -- --csvraw=results.csv unicode-transforms
+# Remove any old benchmark results file first
+$ rm results.csv
+$ cabal run bench -- --csvraw=results.csv --quick
 ```
 
+It will collect the benchmark results in `results.csv` file.
+
 Now repeat the above step for the new code i.e. with the changes.
-It will append the benchmarks results to the existing csv file.
+It will append the new benchmarks results to the existing `results.csv` file.
 
-If you want the benchmarks to run quickly for a quick overview then use
-`--quick` option when running the benchmarking executable.
+If you want more accurate benchmark results remove the `--quick` flag.
 
-Build the bench-show reporting executable `chart`.
+To generate a benchmark comparison between old and new changes from the
+benchmark results in `results.csv` file:
 
 ```
 # [NOTE] The path "results.csv" is harcoded in the chart executable.
-$ cabal v2-run chart
+$ cabal run chart --flag bench-show
+```
+
+# Comparing with ICU (text-icu package)
+
+Install the icu library:
+
+```
+# On Mac OS using MacPorts
+$ sudo port install icu
```

Comment:
Alternatively `brew install icu4c`

Reply:
I use macports, I know that works. I can add the brew example as well if you know it works. BTW why is it `icu4c` with brew and not just `icu`?

Reply:
I checked it yesterday, `brew install icu4s` works fine for me. 
(I do not know why it is not just `icu` %)

Reply:
Can you confirm if it is `icu4s` or `icu4c`? Earlier you wrote `icu4c` in all the command examples.